docker/fpm/dev.ini

```diff
@@ -1,6 +1,6 @@
 date.timezone = Europe/Helsinki
 intl.default_locale = fi
-memory_limit = 256M
+memory_limit = 1024M
```

Comment:
Why? 256M is plenty

Reply:
These weren't supposed to be committed. 🥲